islandora_solr.install

```diff
@@ -328,7 +330,6 @@ function islandora_solr_search_settings_variables() {
     'islandora_solr_primary_display',
     'islandora_solr_request_handler',
     'islandora_solr_luke_timeout',
-    'islandora_solr_breadcrumbs_parent_fields',
```

Comment:
Whoops... Yeah, I did a goof.

... We should probably break the forms up into separate files, at some point.